jsk_pcl_ros/include/jsk_pcl_ros/cluster_point_indices_decomposer.h

```diff
@@ -173,6 +173,7 @@ namespace jsk_pcl_ros
     bool force_to_flip_z_axis_;
     bool publish_clouds_;
     bool publish_tf_;
+    bool _publish_tf_;
```

Comment:
remove this. you can set `bool publish_tf` as local variable

Reply:
Well,  `_publish_tf_` is used in both `bool ClusterPointIndicesDecomposer::computeCenterAndBoundingBox` and `void ClusterPointIndicesDecomposer::extract`, so we have to pass parameters to a function either by pointers or by reference. 
I don't know if this method is appropriate.

Reply:
then, pass it as a function arguments.

Reply:
Ok, I done it.